pxr/usd/usdUtils/testenv/testUsdUtilsValidators.cpp

```diff
@@ -0,0 +1,102 @@
+#include "pxr/usd/usd/validator.h"
+#include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usdUtils/validatorTokens.h"
+#include "pxr/usd/usd/validationRegistry.h"
+#include "pxr/base/arch/systemInfo.h"
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+static
+void
+TestUsdUsdzValidators()
+{
+    // This should be updated with every new validator added with
+    // UsdUtilsValidators keyword.
+    UsdValidationRegistry& registry = UsdValidationRegistry::GetInstance();
+    UsdValidatorMetadataVector metadata =
+            registry.GetValidatorMetadataForKeyword(
+                    UsdUtilsValidatorKeywordTokens->UsdUtilsValidators);
+    // Since other validators can be registered with a UsdUtilsValidators
+    // keyword, our validators registered in usd are a subset of the entire
+    // set.
+    std::set<TfToken> validatorMetadataNameSet;
+    for (const UsdValidatorMetadata &metadata : metadata) {
+        validatorMetadataNameSet.insert(metadata.name);
+    }
+
+    const std::set<TfToken> expectedValidatorNames =
+            {UsdUtilsValidatorNameTokens->packageEncapsulationValidator};
+
+    TF_AXIOM(std::includes(validatorMetadataNameSet.begin(),
+                           validatorMetadataNameSet.end(),
+                           expectedValidatorNames.begin(),
+                           expectedValidatorNames.end()));
+}
+
+static
+void
+TestPackageEncapsulationValidator()
+{
+    UsdValidationRegistry& registry = UsdValidationRegistry::GetInstance();
+
+    // Verify the validator exists
+    const UsdValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdUtilsValidatorNameTokens->packageEncapsulationValidator);
+
+    TF_AXIOM(validator);
+
+    // Load the pre-created usdz stage with paths to a layer and asset
+    // that are not included in the package, but exist
+    const UsdStageRefPtr& stage = UsdStage::Open(
+            "testUsdUtilsPackageEncapsulation/fail.usdz");
```

Comment:
The assets are being "installed" in the test dir, so the file here shouldn't include the test dir. Just "fail.usdz" should pick the asset. This is the reason for test failures.